src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,39 @@
+package calculator;
+
+import java.util.List;
+
+public class Numbers {
+
+    private final List<Integer> numbers;
+
+    public Numbers(List<String> contents) {
+        this.numbers = toInteger(contents);
+        validateNonNegative();
+    }
```

Comment:
```suggestion
    public Numbers(List<String> contents) {
        this(toInteger(contents));
    }
    
    public Numbers(List<Integer> numbers) {
        validateNonNegative(numbers);
        this.numbers = numbers;
    }
```

위와 같이 Integer 타입을 가지는 또 다른 생성자를 오버라이딩하는 것은 어떨까요?